src/commands/Minion/defaultfarming.ts

```diff
@@ -51,6 +51,13 @@ export default class extends BotCommand {
 	async tier(msg: KlasaMessage, [newCompostTier]: [CompostTier]) {
 		await msg.author.settings.sync(true);
 
+		if (newCompostTier === undefined) {
+			return msg.send(
+				'You must specify a valid compost type. The available tiers to select are `compost`, `supercompost`, and `ultracompost`.' +
+					`For example, \`${msg.cmdPrefix}defaultfarming tier supercompost\`.`
```

Comment:
This is gonna have a weird lack of a space after the fullstop, e.g. `ultracompost.For example`